src/api/customAxiosInstance.js

```diff
@@ -17,9 +17,9 @@ import hathorLib from '@hathor/wallet-lib';
  * Create an axios instance to be used when sending requests
  *
  * @param {callback} resolve Callback to be stored and used in case of a retry after a fail
- * @param {number} timeout Timeout in milliseconds for the request
+ * @param {number} [_timeout] Timeout in milliseconds for the request
  */
-const createRequestInstance = (resolve, timeout) => {
+const createRequestInstance = (resolve, _timeout) => {
```

Comment:
suggestion(if-minor): Should we remove timeout?

If its not used we could just remove this argument, right?

Reply:
Good point. Removed on https://github.com/HathorNetwork/hathor-explorer/pull/294/commits/f38cf8ecff200cf9b0fb4764fc167158f13ac1e7